app/styles/ilios-common/components/single-event.scss

```diff
@@ -8,6 +8,7 @@
 
   h2 {
     @include ilios-heading-h4;
+    display: inline;
```

Comment:
Instead I think the icon should go inside the `h2`, otherwise it's just hanging out on it's own and will break weirdly on smaller screens.

Reply:
👍